src/lib/CMakeLists.txt

```diff
@@ -347,6 +347,23 @@ set(
     ${TBB_LIBRARY}
 )
 
+if (NOT DISABLE_JIT_SUPPORT)
```

Comment:
Shouldn't you check for `LLVM_FOUND` here as well?

Reply:
`DISABLE_JIT_SUPPORT` is set to `ON` [here](https://github.com/hyrise/hyrise/pull/696/files#diff-95e351a3805a1dafa85bf20b81d086e6R9), if either of a number of conditions is not met. All subsequent checks then only check this flag instead of repeating all the conditions

Reply:
I asked because it is inconsistent with `if (NOT DISABLE_NUMA_SUPPORT AND ${NUMA_FOUND})`, but this is more a problem with what NUMA does.